README.rst

```diff
@@ -66,10 +66,17 @@ These samples can be used to compute any downstream estimates of interest.
 .. code-block:: python
 
     y_samples = model.sample(x, n_samples=100, verbose=True)
+    # y_samples.shape == (n_samples, x.shape[0], y.shape[1])
```

Comment:
Nitpick but rather than a comment could you make it an assert statement?
```python
    y_samples = model.sample(x, n_samples=100, verbose=True)
    assert y_samples.shape == (n_samples, x.shape[0], y.shape[1])
```
 I feel the comment underneath looks weird hahaha. I mostly see comments to the side or up.

Reply:
i don;t like assert though. in fact the assert would fail haha because the shape is wrong. that is because of our "user-friendly" handling of shape of y. in the readme, y is squeezed so the y_samples will also be squeezed.
the correct one is:
```suggestion
    y_samples = model.sample(x, n_samples=100, verbose=True) # shape (n_samples, x.shape[0])
```
but it is wrong in other cases. do you think I should just remove the comment?

Reply:
Ah! I forgot about this. Yeah then I think we should probably remove the comment or add the correct one. Not sure how to express this with a simple statement.